### DISCUSSION TITLE
Redash with Digital Ocean Application Platform

### DISCUSSION THREAD

**Discussion by naouflex** [getredash/redash#5922] [Q&A]
Hello community,

I am struggling to implement a redash instance using the Application Platform on Digital Ocean. I want to implement it this way because it allows to set up a dev pipeline with my fork in github and customize further the frontend, I don't want to rely on the Droplet version offered in the marketplace.

Anyone would be aware of a successful implementation this way that I could take inspiration from the config ?

Have a nice day !

**Comment by arikfr** [comment]
Can you provide some more details on how you set it up on the Application Platform?

**Reply by naouflex** [reply]
Many thanks for your answer Arik.

I have actually just solved it after some iterations. In case anyone is looking for the same answer :

> Remove additional worker created by Digital Ocean automatically on port 8080
> Create postgresql and redis database instances ( you can attach it or not to the same App as a resource)
> Add the following to your env variables :
     - REDASH_COOKIE_SECRET=<cookie_cookie_cookiiiiie_seriously_use_a_key>
     - REDIS_URL=rediss://default:password@db_url:db_port
     - DATABASE_URL=postgresql://doadmin:password@db_url:25060/defaultdb
> Build & deploy
> Connect to console and initialize the db : ./manage.py database create_tables
> Good to go ! Connect to the url and set up your admin account 👍

**Comment by naouflex** [comment]
> Remove additional worker created by Digital Ocean automatically on port 8080
>   Create postgresql database instance ( you can attach it or not to the same App as a resource)
>    Add the following to your env variables :
    - REDASH_COOKIE_SECRET=<cookie_cookie_cookiiiiie_seriously_use_a_key>
    - DATABASE_URL=postgresql://doadmin:password@db_url:25060/defaultdb
    Build & deploy
    Connect to console and initialize the db : ./manage.py database create_tables
    Good to go ! Connect to the url and set up your admin account 👍

**Reply by arikfr** [reply]
What about Redis?
